client/src/components/Workflow/WorkflowInvocationsCount.vue

```diff
@@ -1,13 +1,13 @@
 <script setup lang="ts">
 import { library } from "@fortawesome/fontawesome-svg-core";
-import { faClock, faSitemap } from "@fortawesome/free-solid-svg-icons";
```

Comment:
guess I wasn't the first one who tried faClock, since it was already in the imports :)